src/TabView.tsx

```diff
@@ -238,39 +260,51 @@ const TabView = <Route extends BaseRoute>({
       barTintColor={barTintColor}
       rippleColor={rippleColor}
     >
-      {trimmedRoutes.map((route) => {
-        if (getLazy({ route }) !== false && !loaded.includes(route.key)) {
-          // Don't render a screen if we've never navigated to it
-          if (Platform.OS === 'android') {
-            return null;
+      <MaybeScreenContainer
+        enabled={detachInactiveScreens}
+        hasTwoStates
+        style={styles.container}
+      >
```

Comment:
Have you tested it on both new and old architectures? I feel like this might break passing children on new arch as technically now we have only one child.

Reply:
Till now I tested this on Android for both new and old arch.
I will be testing it on iOS today.